tests/test_documents.py

```diff
@@ -645,3 +645,13 @@ def test_get_document_internal_stats(user: UserClient, user2: UserClient):
     export_csv = list(obj_csv)
 
     assert csv_str.strip() == '""'
+
```

Comment:
@cayop Do you think it is necessary to add a test to check the login / logout and then check if token is valid after the logout?

A similar test may already exist, at least the login test alone.
As you think is best.